controllers/numaresourcesoperator_controller.go

```diff
@@ -143,9 +145,7 @@ func (r *NUMAResourcesOperatorReconciler) Reconcile(ctx context.Context, req ctr
 	if condition != "" {
 		// TODO: use proper reason
 		reason, message := condition, messageFromError(err)
-		if err := status.Update(ctx, r.Client, instance, condition, reason, message); err != nil {
-			klog.InfoS("Failed to update numaresourcesoperator status", "Desired condition", condition, "error", err)
-		}
+		_, _ = r.updateStatus(ctx, instance, condition, reason, message)
```

Comment:
this emits the event AND also logs, right?

Reply:
Yes.
But following the next comment of yours, I'll probably remove the event and keep the log